clmm/modeling.py

```diff
@@ -417,10 +424,10 @@ def predict_tangential_shear(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo,
     delta_sigma = predict_excess_surface_density(r_proj, mdelta, cdelta, z_cluster, cosmo,
                                                  delta_mdef=delta_mdef,
                                                  halo_profile_model=halo_profile_model)
-
     if z_src_model == 'single_plane':
         sigma_c = get_critical_surface_density(cosmo, z_cluster, z_source)
-        gammat = delta_sigma / sigma_c
+        gammat = np.nan_to_num(delta_sigma / sigma_c)
```

Comment:
Explicitly add:
`gammat = np.nan_to_num(delta_sigma / sigma_c, nan=0.0)`

Reply:
Quick question - I think sigma_c can be 0, so gammat can also be infinity.  In this case, the default for nan_to_num is to make that some "large number".  Should we explicitly assign what nan_to_num does with infinity?